py/torch_tensorrt/dynamo/conversion/impl/activation/ops.py

```diff
@@ -0,0 +1,349 @@
+
+    def tanh_dyn_range_fn(dyn_range):
+        def tanh_fn(x):
+            return (np.exp(x) - np.exp(-x)) / (np.exp(x) + np.exp(-x))
+
+        return tanh_fn(dyn_range[0]), tanh_fn(dyn_range[1])
+
+    return convert_activation(
+        network,
+        target,
+        source_ir,
+        name,
+        operation_type,
+        input_val,
+        dyn_range_fn=tanh_dyn_range_fn,
+    )
+
+
+def leaky_relu(
+    network: TRTNetwork,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input_val: TRTTensor,
+    alpha: Optional[Any] = 0.01,
+):
+    operation_type = trt.ActivationType.LEAKY_RELU
+
+    def leaky_relu_dyn_range_fn(dyn_range):
+        def leaky_relu_fn(x):
+            return max(0, x) + alpha * min(0, x)
+
+        return leaky_relu_fn(dyn_range[0]), leaky_relu_fn(dyn_range[1])
+
+    return convert_activation(
+        network,
+        target,
+        source_ir,
+        name,
+        operation_type,
+        input_val,
+        alpha,
+        dyn_range_fn=leaky_relu_dyn_range_fn,
+    )
+
+
+def elu(
+    network: TRTNetwork,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input_val: TRTTensor,
+    alpha: Optional[Any] = 1.0,
+    beta: Optional[Any] = None,
+):
+    EPS = 1e-4
+    # actually call selu()
+    if (
+        abs(alpha - 1.6732632423543772) < EPS
+        and beta is not None
+        and abs(beta - 1.0507009873554805) < EPS
+    ):
+        print("Selu is called but re-uses elu function!")
+        return selu(network, target, source_ir, name, input_val)
+
+    else:
+        operation_type = trt.ActivationType.ELU
+
+        def elu_dyn_range_fn(dyn_range):
+            return (
+                torch.nn.ELU(dyn_range[0], alpha),
+                torch.nn.ELU(dyn_range[1], alpha),
+            )
+
+        return convert_activation(
+            network,
+            target,
+            source_ir,
+            name,
+            operation_type,
+            input_val,
+            alpha,
+            dyn_range_fn=elu_dyn_range_fn,
+        )
+
+
+def selu(
+    network: TRTNetwork,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input_val: TRTTensor,
+):
+    operation_type = trt.ActivationType.SELU
+
+    def selu_dyn_range_fn(dyn_range):
+        return (torch.nn.SELU(dyn_range[0]), torch.nn.SELU(dyn_range[1]))
+
+    return convert_activation(
+        network,
+        target,
+        source_ir,
+        name,
+        operation_type,
+        input_val,
+        dyn_range_fn=selu_dyn_range_fn,
+    )
+
+
+# no corresponding function in aten/native_functions
+def softsign(
+    network: TRTNetwork,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input_val: TRTTensor,
+):
+    operation_type = trt.ActivationType.SOFTSIGN
+
+    def softsign_dyn_range_fn(dyn_range):
+        return (torch.nn.Softsign(dyn_range[0]), torch.nn.Softsign(dyn_range[1]))
```

Comment:
Switch to [`torch.nn.functional.softsign`](https://pytorch.org/docs/stable/generated/torch.nn.functional.softsign.html?highlight=softsign#torch.nn.functional.softsign), to use functional implementation.